Extensions/Firebase/firebasetools.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Firebase Tools
+ * @author arthuro555
+ */
+
+/*    Initialization of the firebase SDK    */
+
+/** A Timeout function is used to initialize firebase as soon as the project data has been loaded */
+let gdjsWatcher = function() {
+    if (gdjs.projectData === undefined) {
+        setTimeout(gdjsWatcher, 1);
+        return;
+    }
+    let firebaseConfig;
+    try { 
+        firebaseConfig = JSON.parse(gdjs.projectData.properties.firebaseConfig);
```

Comment:
Prefer using [`runtimeScene.getGame().getGameData()`](https://docs.gdevelop-app.com/GDJS%20Runtime%20Documentation/gdjs.RuntimeGame.html#getGameData).

Reply:
I would like but I don't have access to runtimeGame here. Should I switch to a first scene loaded callback?